### DISCUSSION TITLE
compare binary contents ignore filenames

### DISCUSSION THREAD

**Discussion by emanueol** [WinMerge/winmerge#1415] [Q&A] [answered]
Is this possible in WinMerge ?

Normally Win Merge compare contents and then shows in a 2 column side by side the differences by filename, but what I need is to ignore filenames and just compare by contents, is this possible ?

Thanks ~

**Comment by sdottaka** [comment, marked as answer]
Unfortunately, there is no such feature.

However, WinMerge can display a column of hash values for files and group files that have matching hash values but different file names or folders. This may help.
![image](https://user-images.githubusercontent.com/98126/180099913-2db68fbc-02a9-4313-8f52-0b3c094e899a.png)

![image](https://user-images.githubusercontent.com/98126/180099999-bbc6712a-f720-46b8-ba7c-17205d9cd3b5.png)
![image](https://user-images.githubusercontent.com/98126/180100050-cd94ac2b-402e-4fb2-b67f-43d06fb33fce.png)
![image](https://user-images.githubusercontent.com/98126/180100131-1e2a1014-8263-4ce0-bb4d-724b50536bee.png)
![image](https://user-images.githubusercontent.com/98126/180100165-9acc5a4c-4c2c-4b62-8db3-ff99079f0cfd.png)

**Reply by emanueol** [reply]
Perfect! Thank you very much!!

**Reply by Gitoffthelawn** [reply]
@sdottaka This is a very clever trick.  Thank you.

**Reply by ashauryn** [reply]
For some reason the  "additional properties" is grayed out for me.  Any suggestions?

**Reply by sdottaka** [reply]
The 32bit version of WinMerge does not support Additional Properties. Please use the 64bit version of WinMerge.
